lib/util.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+const dotenv   = require('dotenv');
+const fs       = require('fs');
+const pug      = require('pug');
+const newrelic = require('newrelic');
+const spawn    = require('child_process').spawn
+
+/**
+ * Read dotenv (.env + ENV)
+ */
+exports.readEnv = () => {
+  let exampleText = fs.readFileSync(`${__dirname}/../env-example`);
+  let exampleKeys = Object.keys(dotenv.parse(exampleText));
+
+  let dots = {};
+  try {
+    let dottext = fs.readFileSync(`${__dirname}/../.env`);
+    dots = dotenv.parse(dottext);
+  } catch (err) {}
+
+  let env = {};
+  for (let key of exampleKeys) {
+    if (process.env[key] !== undefined || (dots[key] && dots[key] !== '')) {
+      let val = process.env[key] || dots[key];
+      if (val === 'true') {
+        env[key] = true;
+      } else if (val === 'false') {
+        env[key] = false;
+      } else if (isNaN(val) || val == '') {
+        env[key] = val;
+      } else {
+        env[key] = parseInt(val);
+      }
+    } else {
+      env[key] = undefined;
+    }
+  }
+  return env;
+};
+
+/**
+ * Find the script tags to include
+ */
+exports.findScripts = (isDist) => {
+  let scripts = ['inline.js'];
+  if (isDist) {
+    let files = [];
+    try { files = fs.readdirSync(`${__dirname}/../dist`); } catch (e) {}
+    scripts.push(files.find(f => f.match(/^styles\..+\.bundle\.js$/)));
```

Comment:
The built `/dist` files have a hash string in them ... so i just scanned the directory to match on them.